test/unit/SourceLocator/Type/EvaledCodeSourceLocatorTest.php

```diff
@@ -9,18 +9,55 @@
 use Roave\BetterReflection\Reflector\Reflector;
 use Roave\BetterReflection\SourceLocator\Type\EvaledCodeSourceLocator;
 use Roave\BetterReflection\SourceLocator\Located\EvaledLocatedSource;
+use Roave\BetterReflection\SourceLocator\Ast\Locator;
 
 /**
  * @covers \Roave\BetterReflection\SourceLocator\Type\EvaledCodeSourceLocator
  */
 class EvaledCodeSourceLocatorTest extends \PHPUnit_Framework_TestCase
 {
     /**
-     * @return Reflector|\PHPUnit_Framework_MockObject_MockObject
+     * @var Reflector
      */
-    private function getMockReflector()
+    private $reflector;
+
+    /**
+     * @var ReflectionClass
+     */
+    private $reflectionClass;
+
+    /**
+     * @var Locator
+     */
+    private $locator;
+
+    public function setUp()
```

Comment:
Have refactored this test slightly - I inject the `Locator` mock into each new instance rather than creating a new test which duplicates another test in order to see if an arbitrary method was called on the newly injected `Locator`, which seems fragile.

As we have more mocks I switched to a `setUp` style for creating the mock objects.

wdyt?